packages/platform/constants/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@calcom/tsconfig/base.json",
+  "compilerOptions": {
+    "target": "es5",
+    "jsx": "preserve",
```

Comment:
For constants packages tsconfig and one of types and utils packages - do we need `"jsx": "preserve"` ?  We would need it if we would use `.tsx` files which probably atoms would be the only place for such files.